paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -873,8 +867,10 @@ internal class CustomerSheetViewModel(
                     customPrimaryButtonUiState = uiState,
                 )
             } else {
+                val enabled = it.paymentMethodCode == USBankAccount.code || it.formFieldValues != null
```

Comment:
Once a bank account is linked, we no longer provide a `customPrimaryButtonUiState` from the US Bank Account component.